scripts/repackage.sh

```diff
@@ -88,7 +88,7 @@ NEWPKGJSONHASH=$(
     }) | to_entries | sort_by(.key) | from_entries,
```

Comment:
Does anything test that repackage.sh does something sensibly? If so, does it still pass that test. If not, I am not suggesting that this PR should be the first to test it. But should it at least have an issue that it should be tested?

Reply:
I think this gets run by @kriskowal regularly

Reply:
@kriskowal ?

In any case, no impediment to this PR so resolving here.

Reply:
I use repackage.sh to scaffold new packages and rescaffold old packages. I’ve never used it without alteration because the needs shift between uses. This is one of those things that I leave around for semiautomatic manual fixes which is not sufficiently automatable to warrant a test.